<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationDialog</class>
 <widget class="QDialog" name="NotificationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>447</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notification</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/sound.png</normaloff>:/icons/sound.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color:rgb(54,54,54);
color:white;
font-size:12px;
}
QGroupBox{
border:1px solid grey;
border-radius:5px;
padding:10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_no">
     <property name="styleSheet">
      <string notr="true">*{
font-size:18px;
border:none;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>No Reminders today!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_debts">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="title">
      <string>Debts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="listView_debts">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_cheque">
     <property name="styleSheet">
      <string notr="true">*{
border:none;
}
QListWidgetItem{
min-height:30px;
padding:1px;
}</string>
     </property>
     <property name="title">
      <string>Cheque Maturity</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listView_cheque">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
        <property name="uniformItemSizes">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
